CONTRIBUTING.md

```diff
@@ -90,7 +90,13 @@ This workflow utilizes a docker container to set up most dependencies ensuring a
       DOCKER_IMAGE_TAG=my_tag ./docker/build_container_dev.sh
       ```
       Would build the container `morpheus:my_tag`.
-   1. Note: This does not build any Morpheus or Neo code and defers building the code until the entire repo can be mounted into a running container. This allows for faster incremental builds during development.
+   1. If you would like to build the container to have a default user, other than root, add the following to your 
+   `build_container_dev.sh` call.
+      ```bash
+      DOCKER_EXTRA_ARGS="--build-arg MORPHEUS_USER=[YOUR_USER_NAME]" ./docker/build_container_dev.sh
+      ```
```

Comment:
I think this is pretty safe to include by default in the `build_container_dev.sh` and omit from `build_container_release.sh` is it not? We could just add the following to `build_container_dev.sh`:

```
DOCKER_EXTRA_ARGS="${DOCKER_EXTRA_ARGS} --build-arg MORPHEUS_USER=$USER"
```

Or better yet, make a new option `DOCKER_IMAGE_USER` in `build_container.sh` which will set the new build arg.

Reply:
Generally speaking, it probably makes sense to specify a user for the release container too, just from a security standpoint, since we do things like mount volumes by default. We could set it to something default, like 'morpheus'. I don't think there is any downside to doing so.